app/code/community/Kirchbergerknorr/UrlCacheControl/Mage/Core/Model/Design/Package.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * @author      Nick Dilßner <nd@kirchbergerknorr.de>
+ * @copyright   Copyright (c) 2018 kirchbergerknorr GmbH (http://www.kirchbergerknorr.de)
+ * @license     http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class Kirchbergerknorr_UrlCacheControl_Mage_Core_Model_Design_Package extends Mage_Core_Model_Design_Package
+{
+    /**
+     * parameter _type
+     */
+    const SKIN = 'skin';
+
+    /**
+     * add modified timestamp to url
+     *
+     * {@inheritdoc}
+     */
+    public function getSkinUrl($file = null, array $params = array())
+    {
+        // <COPY - optimized with getFilename() (contains updateParamDefaults(), but without referenced params - that means all changes happens in parent, except 'type')
```

Comment:
Please consider using our '<kk ... kk>' tag convention (see Confluence) for things you changed within copied sections.

Reply:
ok... added my opinion in confluence, feel free to answer the question